src/components/App.jsx

```diff
@@ -4,6 +4,7 @@ import DefaultLayout from '../layouts/default.layout.jsx';
 import MainLayout from '../layouts/main.layout.jsx';
 
 class App extends Component {
+	
 	render() {
 		return (
 			<div className='app'>
```

Comment:
Just like what @Jajabenit250 said, there is no need to have /auth route, once I visit localhost://8080, it should just redirect me to the login page where you have the social icon buttons and @Jajabenit250 would add the remaining elements to the page.

Reply:
@Ruqoyah, you mean the login should be like a landing page?

Reply:
I checked through the mockup on Figma and found out the landing page is the login page (I hope I am correct) so I am saying if I visit localhost://8080, it should redirect me to localhost://8080/login

Reply:
No, we don't actually have a dashboard, we started working on signup and login. login is not the dashboard